packages/react-core/src/components/ContextSelector/ContextSelectorToggle.tsx

```diff
@@ -104,7 +107,13 @@ export class ContextSelectorToggle extends React.Component<ContextSelectorToggle
         {...props}
         id={id}
         ref={this.toggle}
-        className={css(styles.contextSelectorToggle, isActive && styles.modifiers.active, className)}
+        className={css(
+          styles.contextSelectorToggle,
+          isActive && styles.modifiers.active,
+          isPlainText && styles.modifiers.plain,
+          isPlainText && styles.modifiers.text,
```

Comment:
Would there ever be a time you'd want just `styles.modifiers.text?`?

Reply:
Not at the moment, cleaned that up to a one liner